### PR TITLE
Mistic100 bootstrap confirmation 4.0.0

### PR DESCRIPTION
**Pull request by theeternalsw0rd**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mistic100/Bootstrap-Confirmation/
 * Official website (optional, not the repository): http://bootstrap-confirmation.js.org/
 * NPM package url (optional): https://www.npmjs.com/package/bootstrap-confirmation2
 * License and its reference: Apache License, Version 2.0 https://www.apache.org/licenses/LICENSE-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 21
   - Count of stars: 245
   - Count of forks: 215
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week: 1762
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] Doesn't already exist / has duplicate issue and PR
 * [checkbox:checked] The lib has notable popularity
   * [checkbox:checked] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [checkbox:checked] More than 800 downloads stats per month on npm registry
 * [checkbox:checked] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [checkbox:checked] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by theeternalsw0rd** [comment]
Sorry, looks like I messed up the checkboxes. First time doing a pull request like this and doing checkboxes in markdown. Even looked up how to do it, but obviously the documentation I found was not correct. If someone wouldn't mind telling me the correct way, I'd appreciate it for future use.

File: ajax/libs/mistic100-Bootstrap-Confirmation/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mistic100-Bootstrap-Confirmation",
   "filename": "bootstrap-confirmation.min.js",
-  "version": "2.4.1",
+  "version": "4.0.0",
   "homepage": "https://mistic100.github.io/Bootstrap-Confirmation",
```
Comment (extend1994) [inline ajax/libs/mistic100-Bootstrap-Confirmation/package.json]: Let's help change this to `http://bootstrap-confirmation.js.org/`

**Comment by theeternalsw0rd** [comment]
Bootstrap 4 has a built-in modal system. I ended up just using that and haven't had time to look at this further. Might get a chance next week.

**Comment by sashberd** [comment]
@MattIPv4 Should we merge this one? It is fork from some main stream, and also it is already included in boostrap 4

**Comment by MattIPv4** [comment]
Seems this has conflicts and is introducing versioned files manually.

**Comment by MattIPv4** [comment]
Hi there,

We are no longer accepting pull requests that manually add versioned files for libraries. Instead, please ensure that the library has an up-to-date auto-update config for either NPM or Git in the cdnjs repository.

You can find out more about our auto-update config structure in the [CONTRIBUTING](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) and [CONTRIBUTING-WIP](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md) documents.

**Pull request closed without merging**